dialogue-clients/src/test/java/com/palantir/dialogue/clients/DialogueDnsResolutionWorkerTest.java

```diff
@@ -144,7 +144,8 @@ public void testResolvedAddressesChangesAfterStartup() throws Exception {
                 .build();
         SettableRefreshable<ServicesConfigBlock> inputRefreshable = Refreshable.create(initialState);
         SettableRefreshable<ServicesConfigBlockWithResolvedHosts> receiverRefreshable = Refreshable.create(null);
-        DialogueDnsResolutionWorker worker = new DialogueDnsResolutionWorker(resolver, receiverRefreshable::update);
+        DialogueDnsResolutionWorker worker =
+                new DialogueDnsResolutionWorker(resolver, Duration.ofSeconds(5), receiverRefreshable);
```

Comment:
I think we may want to externally trigger state changes in this tests dns resolver, rather than leveraging interactions to change the state.

I don't want to do that in this PR though, since it's already complex enough